src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -307,3 +313,89 @@ def build_sdfg_from_itir(
         sdfg.save(sdfg_filename)
 
     return sdfg
+
+
+@dataclasses.dataclass(frozen=True)
+class Program(Program, SDFGConvertible):  # type: ignore[no-redef]
+    """Extension of GT4Py Program : Implement SDFGConvertible interface"""
+
+    sdfg_convertible: dict[str, Any] = field(default_factory=dict)
```

Comment:
Can you explain the name of this attribute? If I understand the code correctly, it looks like something along the lines of  `sdfg_closure_vars` would be more appropriate... or am I missing something?